Week2/homework/exercise-1.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+/*Exercise 1: Remove the comma
+
+Consider the following string:
+
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+Add the variable to your file.
+Log the length of myString.
+The commas make that the sentence is quite hard to read. Find a way to remove the commas from the string and replace them with spaces. (use Google!)
+After replacing the commas, log myString to see if you succeeded.*/
+
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+console.log('The length of mystring is ' + myString.length);
+myString = myString.split(',');
+myString = myString.join(' ');
+console.log(myString);
+
+/*we can also use replace() method;
+myString = myString.replace(/,\s?/g, ' ');
+console.log(myString);
```

Comment:
👍 but I don't think you need to have `\s?` in the `replace` method. It should also work without it.

Reply:
> Hi @Serafettin99 - I have reviewed your homework and am happy to approve the same. Good luck!

Thanks @yash-kapila for review and comments